### PR TITLE
[RTM] RF: Use N4BiasFieldCorrection(copy_header=True)

### PR DESCRIPTION
**Pull request by effigies**
Taking advantage of nipy/nipype#2034 to simplify workflows slightly.

### CODE REVIEW

File: fmriprep/workflows/fieldmap/phdiff.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def _pick1st(inlist):
     magmrg = pe.Node(IntraModalMerge(), name='magmrg')
 
     # de-gradient the fields ("bias/illumination artifact")
-    n4 = pe.Node(ants.N4BiasFieldCorrection(dimension=3), name='n4')
+    n4 = pe.Node(ants.N4BiasFieldCorrection(dimension=3, copy_header=True), name='n4')
```
Comment (effigies) [inline fmriprep/workflows/fieldmap/phdiff.py]: Note that this wasn't using `init_n4bias_wf` already. If there's a good reason for not copying the header, let me know and I'll revert this.

**Comment by effigies** [comment]
This is failing a 4GB memory limit. ([Memory usage](https://457-78558509-gh.circle-artifacts.com/0/tmp/memory-usage.txt))

I'm not sure why this would newly be an issue. Could be the type of the bias-corrected image (which is now restored to that of the input T1w image). Should I re-run, or is there something obvious to fix up?

**Comment by oesteban** [comment]
This is very weird. Additionally, the process crashing is antsRegistration, not the bias correction.

**Comment by chrisfilo** [comment]
Let's try running first.

**Comment by effigies** [comment]
Rerunning. Also, there's a workflow in niworkflows that should be updated along with this batch.

Blocking merge for poldracklab/niworkflows#173.

**Comment by effigies** [comment]
That one worked. Updating pins and rebasing for a final test.

File: fmriprep/workflows/anatomical.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def init_anat_preproc_wf(skull_strip_ants, output_spaces, template, debug, frees
         RobustMNINormalizationRPT(
             generate_report=True,
             num_threads=omp_nthreads,
-            testing=debug,
+            flavor='testing' if debug else 'precise',
```
Comment (effigies) [inline fmriprep/workflows/anatomical.py]: @oesteban I believe this is the 0.0.7+ equivalent?

Comment (oesteban) [inline fmriprep/workflows/anatomical.py]: yes

**Pull request merged**